components/post_view/reaction/reaction.jsx

```diff
@@ -103,10 +167,13 @@ export default class Reaction extends React.PureComponent {
         if (!this.props.emojiImageUrl) {
             return null;
         }
-
         const {currentUserReacted, users} = this.props.sortedUsers;
+        const {otherUsersCount, canAddReaction, canRemoveReaction} = this.props;
+        const {unreactedNumber, reactedNumber, displayNumber} = this.state;
+        const unreacted = (unreactedNumber > 0) ? unreactedNumber : '';
```

Comment:
Why have these default to the empty string instead of 0?

Also, it might be easier to follow this if these ternaries were inline in the JSX below, although 0/5 about that.

Reply:
The empty string is so the animation doesn’t display a zero when removing a reaction. All aesthetics. I kept this logic up here just to keep the html elements readable below.